packages/generators/init-template/default/webpack.configjs.tpl

```diff
@@ -68,3 +64,25 @@ module.exports = {
         extensions: ['.tsx', '.ts', '.js'],
     },<% } %>
 };
+
+// Add mode specific configurations below
+// Learn more about mode at https://webpack.js.org/configuration/mode/
+
+const production = {
+    mode: 'production',
+};
+
+const development = {
+    mode: 'development',<% if (devServer) { %>
+    devServer: {
+        open: true,
+        host: 'localhost',
+    },<% } %>
+};
+
+module.exports = function (env, argv) {
+    if (argv.mode == 'production') {
+        return merge(base, production);
+    }
+    return merge(base, development);
+};
```

Comment:
I think we can simplify this now, we set `process.env.NODE_ENV` so we can change configuration based on in, `webpack-merge` is good package for multi compiler modes, but I think for fast start we don't need extra packages now, `process.env._NODE_ENV` is enough

Reply:
Okay, looking into it.

Reply:
Done, Thanks.